### DISCUSSION TITLE
Popup message

### DISCUSSION THREAD

**Discussion by B16f00t** [Akascape/CTkMessagebox#7] [Ideas]
Something like this, it'd be a good feature.
 sg.popup_notify("MESSAGE", location=sg.DEFAULT_WINDOW_LOCATION, display_duration_in_ms=300, fade_in_duration=100)

**Comment by Akascape** [comment]
@B16f00t You can add `after` method to automatically remove the messagebox.
And please update the package, I just implemented a fade-in animation parameter.

```python
ctkmessagebox = CTkMessagebox(root_window_location, fade_in_duration=100)
ctkmessagebox.after(1000, ctkmessagebox.button_event) 
```

**Comment by B16f00t** [comment]
It Works Perfect, thanks.
